roles/katello_repositories/tasks/staging_repos.yml

```diff
@@ -53,7 +53,20 @@
     enabled: yes
     include: /etc/yum/foreman.conf
   when:
-    - katello_repositories_version == "nightly" or katello_repositories_version is version('3.15', '>=')
+    - katello_repositories_version != 'nightly'
+    - katello_repositories_version is version('3.15', '>=')
+    - katello_repositories_version is version('3.17', '<=')
+
+- name: 'Add Pulpcore repository'
+  yum_repository:
+    name: pulpcore-repository
+    description: Pulpcore repository for Katello {{ katello_repositories_version }}
+    baseurl: "https://yum.theforeman.org/pulpcore/{{ katello_repositories_pulpcore_version }}/el{{ ansible_distribution_major_version }}/x86_64/"
```

Comment:
This doesn't look like the staging URL Is that intentional? Should there be some `pulpcore-release` package inside that repo itself?

Reply:
There is no staging as such (well, besides repoclosure). I can add such a package, yeah.